index.js

```diff
@@ -71,8 +71,8 @@ function getOptions(uri, o, method) {
         }
     }
 
-    if ((o.method === 'get' || o.method === 'put')
-        && o.retries === undefined) {
+    if ((o.method === 'get' || o.method === 'put') &&
+        o.retries === undefined) {
```

Comment:
One more indent for better readability please

Reply:
Actually fits on 1 line